opencog/atomspace/AtomTable.h

```diff
@@ -93,7 +93,7 @@ class AtomTable
     // handle uuid and the actual atom pointer (since they are stored
     // together).  To some degree, this info is duplicated in the Node
     // and LinkIndex below; we have this here for convenience.
-    std::set<Handle, handle_less> _atom_set;
+    std::unordered_set<Handle, handle_hash> _atom_set;
 
```

Comment:
As a general rule, it would be better to put unrelated changes into different pull requests.  I don't mind the benchmark changes, but will have to explore this one a bit.